go/libraries/doltcore/sqle/writer/noms_table_writer.go

```diff
@@ -56,14 +55,15 @@ type nomsTableWriter struct {
 	tableName   string
 	dbName      string
 	sch         schema.Schema
-	autoIncCol  schema.Column
 	vrw         types.ValueReadWriter
 	kvToSQLRow  *index.KVToSqlRowConverter
 	tableEditor editor.TableEditor
 	sess        WriteSession
-	aiTracker   globalstate.AutoIncrementTracker
 	batched     bool
 
+	autoInc globalstate.AutoIncrementTracker
+	autoOrd int
```

Comment:
What is this if there is no auto inc column? -1? Document this

Reply:
it was unused, deleted it